link/kprobe.go

```diff
@@ -386,6 +412,9 @@ func createTraceFSProbeEvent(typ probeType, args probeArgs) error {
 		// the eBPF program itself.
 		// See Documentation/kprobes.txt for more details.
 		pe = fmt.Sprintf("%s:%s/%s %s", probePrefix(args.ret), args.group, args.symbol, args.symbol)
+		if args.offset != 0 {
```

Comment:
can you put this login in a function similar to `uprobeToken` -> `kprobeToken(symbol, offset)` ?

Reply:
Thanks! I will send v2